c7n/resources/lex.py

```diff
@@ -40,6 +42,90 @@ class resource_type(query.TypeInfo):
     source_mapping = {"describe": query.DescribeWithResourceTags, "config": query.ConfigSource}
 
 
+class LexV2BotAliasDescribe(query.ChildDescribeSource):
+    def augment(self, resources):
+        client = local_session(self.manager.session_factory).client('lexv2-models')
+        for r in resources:
+            botalias = client.describe_bot_alias(
```

Comment:
Should we maybe filter on `botAliasStatus` here? Looks like status could be 'Creating'|'Available'|'Deleting'|'Failed'. We may not be able to apply actions to anything that's not in Available status

Reply:
Lets also remove ResponseMetadata from the API response to eliminate unnecessary data